### DISCUSSION TITLE
iron not recognised by computer

### DISCUSSION THREAD

**Discussion by doubaz222** [Ralim/IronOS#1095] [Q&A] [answered]
So I flashed the firmware all good and everything was running fine but when I went to plug it back in to my computer ( same known working cable, Mac) the iron will no longer show up on my computer.

is there a hidden mode to get it to do this?

I have tried 4 different known working data cables( including the one I originally used to flash the firmware) and 2 different computers but the iron wont go into the "computer mode" it just thinks it's being given a power supply.

Then tried resetting to factory settings with still no luck.

firmware-v2.15.40087E6 09-05-21

please help! :)

**Comment by Ralim** [comment, marked as answer]
As noted on the main page of this firmware; there is **NO** computer mode.
There is only DFU update (but holding the button) or soldering mode.
All settings are in the menu, and the boot logo upload works via using the DFU mode.

**Reply by doubaz222** [reply]
Once again thanks for saving my bacon! What a cool community :)
